extensions/ohif-gradienthealth-extension/src/DicomJSONDataSource/index.js

```diff
@@ -206,6 +209,94 @@ const findStudies = (key, value) => {
   return studies;
 };
 
+const mapSegSeriesFromDataSet = (dataSet) => {
+  return {
+    Modality: dataSet.Modality,
+    SeriesInstanceUID: dataSet.SeriesInstanceUID,
+    SeriesDescription: dataSet.SeriesDescription,
+    SeriesNumber: Number(dataSet.SeriesNumber),
+    SeriesDate: dataSet.SeriesDate,
+    SliceThickness:
+      Number(dataSet.SharedFunctionalGroupsSequence.PixelMeasuresSequence
+        .SliceThickness),
```

Comment:
Are you sure you don't want to use the top level slice thickness number? tag `(0018,0050)`
https://dicom.innolitics.com/ciods/rt-dose/image-plane/00180050

Reply:
Already removed